contracts/Rewarder.sol

```diff
@@ -278,43 +292,57 @@ contract Rewarder is IRewarder, Pausable {
         );
 
         ITaxManager taxManager = getTaxManager();
+
         uint256 taxRateDivisor = taxManager.taxBaseDivisor();
         address seekerHandler = nexus.getHandler(_seekerId);
-        emit seekerTaxPayedToken(seekerHandler, msg.sender,  _platformTax + _referralTax, token);
+        address solverHandler = nexus.getHandler(_solverId);
+
+        emit SeekerTaxPaidToken(seekerHandler, msg.sender,  _platformTax + _referralTax, token);
         // Seeker tax distribution
 
         // Platform tax distribution
-        address platformTaxReceiver = taxManager.platformTaxReceiver();
+        address platformTaxReceiver = taxManager.platformRevenuePool();
```

Comment:
we reward seeker referrals (they brought person that spends money to the platform)